templates/docs/examples/patterns/chip/link.html

```diff
@@ -0,0 +1,111 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Chip / Link{% endblock %}
+
+{% block standalone_css %}patterns_chip{% endblock %}
+
+{% block content %}
+  <a href="#" class="p-chip">
+    <span class="p-chip__lead">Owner</span>
+    <span class="p-chip__value">Bob</span>
+  </a>
+
+  <a href="#" class="p-chip">
+    <span class="p-chip__value">21.10</span>
+  </a>
+
+  <a href="#" class="p-chip" aria-pressed="true">
+    <span class="p-chip__value">Selected</span>
+  </a>
+
+  <a href="#" class="p-chip">
+    <span class="p-chip__lead">Owner</span>
+    <span class="p-chip__value">Bob</span>
+    <button class="p-chip__dismiss">Dismiss</button>
```

Comment:
I don't think we should allow this. Having nested interactive/clickable elements leads to confusing UX and accessibility issues.

With button chips we don't do that. If chip is dismissable, it's a span with a button inside it. It should not be interactive as a whole in such case.
I don't think that link chips are any different in this regards.

I guess nothing in the code/styling needs to change, but we should remove examples that use buttons inside links

Reply:
Fixed